<ui version="4.0" >
 <class>UploadProfileDlg</class>
 <widget class="QWidget" name="UploadProfileDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>191</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Profile &amp;name:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>lineProfileName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="KLineEdit" name="lineProfileName" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="text" >
      <string>&amp;Protocol:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>comboProtocol</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QComboBox" name="comboProtocol" />
     </item>
     <item>
      <widget class="QLabel" name="TextLabel4" >
       <property name="text" >
        <string>&amp;Host:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>lineHost</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineHost" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel1_2" >
       <property name="text" >
        <string>Po&amp;rt:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>port</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="port" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="TextLabel2" >
     <property name="text" >
      <string>&amp;User:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>lineUser</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="lineUser" />
   </item>
   <item row="2" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>111</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="TextLabel2_2" >
     <property name="text" >
      <string>Pa&amp;th:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>linePath</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="linePath" />
     </item>
     <item>
      <widget class="QPushButton" name="browseButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QCheckBox" name="defaultProfile" >
     <property name="text" >
      <string>Use as &amp;default profile</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineProfileName</tabstop>
  <tabstop>comboProtocol</tabstop>
  <tabstop>lineHost</tabstop>
  <tabstop>port</tabstop>
  <tabstop>lineUser</tabstop>
  <tabstop>linePath</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>defaultProfile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
